vignettes/basics.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+title: "Basics of shiny.semantic"
```

Comment:
I think we could come up with a better title (SEO reasons). Understanding the basics of shiny.semantic and reasons to use instead of default Shiny Bootstrap.